### PR TITLE
Fix routes

### PR DESCRIPTION
**Pull request by tomsmalley**

### CODE REVIEW

File: frontend/src/Frontend/Storage.hs [hunk 1]
```diff
@@ -115,6 +115,9 @@ class HasStorage m where
 instance Monad m => HasStorage (StorageT m) where
   askStorage = StorageT ask
 
+instance (HasStorage m, Monad m) => HasStorage (RoutedT t r m) where
+  askStorage = lift askStorage
```
Comment (alexfmpe) [inline frontend/src/Frontend/Storage.hs]: https://github.com/kadena-io/chainweaver/blob/c5b763b15500351292c910ff58291c928ebf1ee1/frontend/src/Frontend/Storage.hs#L112

File: frontend/src/Frontend/ReplGhcjs.hs [hunk 2]
```diff
@@ -142,7 +150,7 @@ walletSidebar sidebarExtra = elAttr "div" ("class" =: "sidebar") $ do
 -- | Get the routes to the icon assets for each route
 routeIcon :: R FrontendRoute -> (Text, Text)
 routeIcon = \case
-  FrontendRoute_Main :/ () -> (static @"img/menu/contracts.png", static @"img/menu/contracts_highlighted.png")
+  FrontendRoute_Contracts :/ Nothing -> (static @"img/menu/contracts.png", static @"img/menu/contracts_highlighted.png")
```
Comment (alexfmpe) [inline frontend/src/Frontend/ReplGhcjs.hs]: Doesn't this yield a warning? We're not handling `FrontendRoute_Contracts :/ Just r`

File: frontend/src/Frontend/Storage.hs (hunk 1 above)
Comment (tomsmalley) [reply frontend/src/Frontend/Storage.hs]: hehe

File: frontend/src/Frontend/ReplGhcjs.hs (hunk 2 above)
Comment (tomsmalley) [reply frontend/src/Frontend/ReplGhcjs.hs]: There's a wildcard further down, but I just stuck wildcards after `:/`, seems to make more sense

**Pull request merged**